rules/autoconsent/dpgmedia-nl.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "dpgmedia-nl",
+  "prehideSelectors": [
+    "#pg-shadow-host"
+  ],
+  "detectCmp": [
+    { "exists": "#pg-shadow-host" }
+  ],
+  "detectPopup": [
+    { "visible": "#pg-shadow-host" }
+  ],
+  "optIn": [
+    { "waitForThenClick": "#pg-shadow-host #pg-accept-btn" }
+  ],
+  "optOut": [
+    { "waitForThenClick": "#pg-shadow-host #pg-configure-btn" },
```

Comment:
`#pg-shadow-host` is a shadowRoot element, so this rule does not work. You need to pass an array of selectors, see the [docs here](https://github.com/duckduckgo/autoconsent?tab=readme-ov-file#element-selectors)

Reply:
Thanks, I was aware about it but didn't notice the breakage.